X/XyceWrapper/build_tarballs.jl

```diff
@@ -38,7 +38,7 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    Dependency("Xyce_jll"),
+    Dependency("Xyce_jll"; compat="~7.4.0"),
```

Comment:
Any specific reason for using `~` and not `^`?

Reply:
Hmm I am always confusing the two. I wanted a caret here, thanks!

Reply:
For the record, the `^` is identical to not putting any symbol at all, that one may be easier to remember :wink: